cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/Setup.java

```diff
@@ -61,9 +61,8 @@
      *
      * @param authenticatedContext the context which already contains the authenticated client
      * @param parameters map of parameters
-     * @throws Exception exception is thrown when the parameters does not meet the desired requirements
      */
-    void validateParameters(AuthenticatedContext authenticatedContext, Map<String, String> parameters) throws Exception;
+    void validateParameters(AuthenticatedContext authenticatedContext, Map<String, String> parameters);
```

Comment:
Why do you remove it? And why don't you remove from _validateFileSystem_?

Reply:
Because `AzureSetup.validateAdlsGen2FileSystem()` and `AzureSetup.validateWasbFileSystem()` are throwing exception